WcaOnRails/spec/controllers/competitions_controller_spec.rb

```diff
@@ -589,15 +596,15 @@
         expect(response).to redirect_to root_url
       end
 
-      it "can change registration open/close of locked competition" do
-        competition.update_attribute(:confirmed, true)
+      # it "can change registration open/close of locked competition" do
```

Comment:
As fixing this test doesn't quite fit on this PR, could you please open an issue about it and add a `# FIXME: ` comment linking to the issue?

Reply:
This is going to be handled as soon as #4333 is merged!